community/kernel/src/main/java/org/neo4j/kernel/impl/storageengine/StorageEngine.java

```diff
@@ -162,7 +162,7 @@
     IndexConfigStore indexConfigStore();
 
     @Deprecated
-    KernelHealth kernelHealth();
+    DatabaseHealth kernelHealth();
```

Comment:
This method _could_ be renamed as well, but I wouldn't let it hold back this PR because hopefully we'll be able to remove this method later.